packages/components/popper/src/popper.ts

```diff
@@ -10,6 +10,7 @@ export const Effect = {
   LIGHT: 'light',
   DARK: 'dark',
 } as const
+export type EffectType = typeof Effect[keyof typeof Effect]
```

Comment:
`EffectType` as same as `PopperEffect`

Reply:
Yeah, I have noticed that. So i just need to replace all `EffectType` with `PopperEffect`?

Reply:
Yep.